src/App.js

```diff
@@ -1,23 +1,25 @@
-import logo from './logo.svg';
-import './App.css';
+import { Routes, Route } from 'react-router-dom';
+import './App.scss';
+import Navigation from './components/Navigation/Navigation.jsx';
+import Header from './components/Header';
+import ArticleList from './components/ArticlesList';
+import { useState } from 'react';
 
 function App() {
+
+  const [articles, setArticles] = useState([]);
```

Comment:
Just because nothing else is using this, i'd probably move this state down into articleList.

id only move it if you find out something else needs the articles.